core/templates/dev/head/pages/about/about.html

```diff
@@ -508,6 +508,9 @@
     .oppia-about-foundation-tab-anchor:hover {
       text-decoration: none;
     }
+    .oppia-about-text-align {
```

Comment:
This is fine. We are trying to move styles that are specific to a particular file into that file so we don't load them everywhere. The trick is to ensure that it really is specific to said file.